src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer.java

```diff
@@ -88,7 +88,7 @@ public void run(RunConfiguration config) throws AppEngineException {
     arguments.addAll(DevAppServerArgs.get("runtime", config.getRuntime()));
     arguments.addAll(DevAppServerArgs.get("allow_skipped_files", config.getAllowSkippedFiles()));
     arguments.addAll(DevAppServerArgs.get("api_port", config.getApiPort()));
-    arguments.addAll(DevAppServerArgs.get("automatic_restart", config.getAutomaticRestart()));
+    arguments.addAll(DevAppServerArgs.get("automatic_restart", config.getAutomaticRestart(), true));
```

Comment:
Actually, since a null automaticRestart value still produces no flag and uses the default (i.e., keeps the previous behaviour), I'm going to punt setting the defaults to another PR.
This PR remains "add support for --name=true/false flags"